intern/openmesheffect/host/intern/properties.c

```diff
@@ -85,86 +94,79 @@ void free_properties(OfxPropertySetStruct *properties) {
   }
 }
 
-void deep_copy_property_set(OfxPropertySetStruct *destination, const OfxPropertySetStruct *source) {
+void deep_copy_property_set(OfxPropertySetStruct *destination, const OfxPropertySetStruct *source)
+{
   init_properties(destination);
   append_properties(destination, source->num_properties);
-  for (int i = 0 ; i < destination->num_properties ; ++i) {
+  for (int i = 0; i < destination->num_properties; ++i) {
     deep_copy_property(destination->properties[i], source->properties[i]);
   }
   destination->context = source->context;
 }
 
-bool check_property_context(OfxPropertySetStruct *propertySet, PropertyType type, const char *property) {
+bool check_property_context(OfxPropertySetStruct *propertySet,
+                            PropertyType type,
+                            const char *property)
+{
   switch (propertySet->context) {
-  case PROP_CTX_MESH_EFFECT:
-    return (
-      (0 == strcmp(property, kOfxMeshEffectPropContext) && type == PROP_TYPE_STRING) ||
-      false
-    );
-  case PROP_CTX_INPUT:
-    return (
-      (0 == strcmp(property, kOfxPropLabel) && type == PROP_TYPE_STRING) ||
-      false
-    );
-  case PROP_CTX_HOST:
-    return (
-      (0 == strcmp(property, kOfxHostPropBeforeMeshReleaseCb) && type == PROP_TYPE_POINTER) ||
-      (0 == strcmp(property, kOfxHostPropBeforeMeshGetCb) && type == PROP_TYPE_POINTER) ||
-      false
-    );
-  case PROP_CTX_MESH:
-    return (
-      (0 == strcmp(property, kOfxMeshPropInternalData) && type == PROP_TYPE_POINTER) ||
-      (0 == strcmp(property, kOfxMeshPropHostHandle)   && type == PROP_TYPE_POINTER) ||
-      (0 == strcmp(property, kOfxMeshPropPointCount)   && type == PROP_TYPE_INT)     ||
-      (0 == strcmp(property, kOfxMeshPropVertexCount)  && type == PROP_TYPE_INT)     ||
-      (0 == strcmp(property, kOfxMeshPropFaceCount)    && type == PROP_TYPE_INT)     ||
-      false
-    );
-  case PROP_CTX_PARAM:
-    return (
-      (0 == strcmp(property, kOfxParamPropType) && type == PROP_TYPE_STRING) ||
-      (0 == strcmp(property, kOfxParamPropScriptName) && type == PROP_TYPE_STRING) ||
-      false
-      );
-  case PROP_CTX_ATTRIB:
-    return (
-      (0 == strcmp(property, kOfxMeshAttribPropData) && type == PROP_TYPE_POINTER) ||
-      (0 == strcmp(property, kOfxMeshAttribPropComponentCount) && type == PROP_TYPE_INT) ||
-      (0 == strcmp(property, kOfxMeshAttribPropType) && type == PROP_TYPE_STRING) ||
-      false
-      );
-  case PROP_CTX_OTHER:
-  default:
-    printf("Warning: PROP_CTX_OTHER is depreciated.\n");
-    return true;
+    case PROP_CTX_MESH_EFFECT:
+      return ((0 == strcmp(property, kOfxMeshEffectPropContext) && type == PROP_TYPE_STRING) ||
+              false);
+    case PROP_CTX_INPUT:
+      return ((0 == strcmp(property, kOfxPropLabel) && type == PROP_TYPE_STRING) || false);
+    case PROP_CTX_HOST:
+      return (
```

Comment:
Why is this case formatted differently compared to others? I'm not really used to clang-format